tvb/simulator/simulator.py

```diff
@@ -744,7 +739,7 @@ def _census_memory_requirement(self):
         try:
             memreq += self.surface.triangles.nbytes * 2
             memreq += self.surface.vertices.nbytes * 2
-            memreq += self.surface.vertex_mapping.nbytes * 4  # vertex_mapping, region_average, region_sum
+            memreq += self.surface.region_mapping.nbytes * self.number_of_nodes * 8. * 4  # region_average, region_sum
```

Comment:
? Shouldn't be `region_mapping.nbytes + n * 8 * 4`?

In any case, memory estimation the simulator is fairly sloppy, I wouldn't spend time on this.